jaeger-core/src/main/java/io/jaegertracing/Configuration.java

```diff
@@ -705,7 +705,7 @@ private static boolean getPropertyAsBool(String name) {
           }
           tracerTagMaps.put(tagValue[0], resolveValue(tagValue[1]));
         } else {
-          log.error("Tracer tag incorrectly formatted: " + tag);
+          log.error("JaegerBaseTracer tag incorrectly formatted: " + tag);
```

Comment:
Should probably be `JaegerTracer` - not the internal base class.

Reply:
Just "Tracer"... Probably the IDE tried to be too smart when refactoring :)